src/settings/cli.rs

```diff
@@ -44,6 +44,15 @@ pub struct General {
     /// Number of worker threads result should be a number between 1 and 32,768 though it is advised to keep this value on the smaller side.
     pub threads_multiplier: usize,
 
+    #[structopt(
+        long,
+        short = "b",
+        default_value = "20",
+        env = "SERVER_MAX_BLOCKING_THREADS"
+    )]
+    /// Maximum number of blocking threads 
+    pub max_blocking_threads: usize,
```

Comment:
Should [`max_blocking_threads`](https://docs.rs/tokio/latest/tokio/runtime/struct.Builder.html#method.max_blocking_threads) of `20` be an acceptable default instead of Tokio's `512`?

Reply:
Ah this is because of the fact we are using `static-web-server` in WASI. spawning 512 threads in WASM is quite expensive so we capped it. Maybe its best to have two defaults, one for WASM and one for x86?

Reply:
Ok, I got it.

> Maybe its best to have two defaults, one for WASM and one for x86?

It can be, what if we come up with some `#[cfg(wasm)]` without touching x86? What do you think?